SmartDeviceLink/SDLListFilesOperation.m

```diff
@@ -59,7 +59,14 @@ - (void)sdl_listFiles {
         NSUInteger bytesAvailable = listFilesResponse.spaceAvailable != nil ? listFilesResponse.spaceAvailable.unsignedIntegerValue : 2000000000;
 
         if (weakSelf.completionHandler != nil) {
-            weakSelf.completionHandler(success, bytesAvailable, fileNames, error);
+            if([response.resultCode isEqualToEnum:SDLResultDisallowed]) {
```

Comment:
Instead of this, just always set the `resultCode` into the error if an error exists.